content/cloud/noops-sql-db.md

```diff
@@ -34,24 +35,21 @@ Once you have access to the private beta, please ensure you have [Spin CLI](./cl
 * Your NoOps SQL Database can hold up 1 GB of data
 
 *Service Limitations*
-* The `--sqlite` flag not supported on `spin cloud deploy`
+* The `--sqlite` flag is not supported on `spin cloud deploy`
 * Fermyon **Cloud** only supports databases named "default" at this time
 
-## How To:
+## How To
 
-### Prepare a Database
+Firstly, if you have not already done so, you will need to [create your first Spin application](https://developer.fermyon.com/spin/quickstart#create-your-first-application). Our blog posts [Announcing Fermyon Cloud’s NoOps SQL Database](https://www.fermyon.com/blog/announcing-noops-sql-db) and [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) each have an example of how to create a NoOps SQLite Storage application. To tell Spin that we want to use SQLite storage, we only need to [grant SQLite permission to a component in the application’s manifest]((https://developer.fermyon.com/spin/sqlite-api-guide#granting-sqlite-database-permissions-to-components)) (the spin.toml file). For example:
 
-You can use the `spin cloud sqlite execute` command to check if a table needs to be created, and if so create one: 
-
-```bash
-$ spin cloud sqlite execute inspirational-pig "CREATE TABLE IF NOT EXISTS todos (id INTEGER PRIMARY KEY AUTOINCREMENT,description TEXT NOT NULL,due_date DATE,starred BOOLEAN DEFAULT 0,is_completed BOOLEAN DEFAULT 0)"
+```
+[component]
+sqlite_databases = ["default"]
 ```
 
->> If you need to look up the name of your database, use the `spin cloud sqlite list` command
-
-### Manage the Database Lifecycle
+### Database Access From Many Components
```

Comment:
I'm a bit puzzled by this title.  The original intent of this section was to explain that the database was created at deployment time and persisted across deployments.  I guess the "how to give access" stuff confused the issue but that was meant to be incidental.  I am not sure what this section is now intending to show.

Reply:
Thanks, this has been updated to suite Cloud private preview context.